src/scribe.js

```diff
@@ -71,8 +72,10 @@ define([
     var TransactionManager = buildTransactionManager(this);
     this.transactionManager = new TransactionManager();
 
-    var UndoManager = buildUndoManager(this);
-    this.undoManager = new UndoManager();
+    if (this.options.shouldUndo) {
+      var UndoManager = buildUndoManager(this);
+      this.undoManager = new UndoManager();
```

Comment:
This will be the case anyway. For example `window.myAwesomeVar === undefined`. Unless I'm missing your point?